contrib/ansible/roles/docker/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: restart docker
+  command: systemctl daemon-reload
+  notify:
+    - restart dockerservice
+
+- name: restart dockerservice
+  service: name=docker state=restarted
```

Comment:
I don't love cascading notifications because they make handler ordering harder to read. In this case, there are only 2, so it's not so bad, but just being pedantic...